apps/site/layouts/Download.tsx

```diff
@@ -10,6 +10,7 @@ import styles from './layouts.module.css';
 const DownloadLayout: FC<PropsWithChildren> = async ({ children }) => {
   const {
     frontmatter: { title, subtitle },
+    // eslint-disable-next-line react-hooks/rules-of-hooks
   } = useClientContext();
```

Comment:
Without this disable:

```plaintext
React Hook "useClientContext" cannot be called in an async function.
eslint[react-hooks/rules-of-hooks](https://reactjs.org/docs/hooks-rules.html)
```

I haven't kept up-to-date with the hullabaloo around async functions and hooks. This function doesn't seem to need to be async to me, but 🤷 I don't want to put forward a potentially risky behavior change in this PR.